docs/modules/ROOT/pages/guides/apollo-federation.adoc

```diff
@@ -120,20 +119,21 @@ const driver = neo4j.driver(
   neo4j.auth.basic("neo4j", process.env.NEO4J_PASSWORD)
 );
 
-const neo4jgraphql = new Neo4jGraphQL({
+const neoSchema = new Neo4jGraphQL({
   typeDefs,
   driver,
 });
 
-neo4jgraphql.getSubgraphSchema().then((schema) => {
-  const server = new ApolloServer({
-    schema,
-  });
+const schema = neoSchema.getSubgraphSchema();
 
-  startStandaloneServer(server, { listen: { port: 4002 } }).then(({ url }) => {
-    console.log(`🚀  Server ready at ${url}`);
-  });
+const server = new ApolloServer({
+  schema,
+  resolvers,
```

Comment:
I think there is another `resolvers` in the `docs/modules/ROOT/pages/guides/apollo-federation.adoc` file that can be removed.

Reply:
https://github.com/neo4j/graphql/pull/3239/commits/6fafd2035b4b34651381c93b2d33d4513a2f6fe1 thanks!